go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/applicationautoscaling-controller
 go 1.14
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.0.5
+	github.com/aws-controllers-k8s/runtime v0.0.6
```

Comment:
Not taking it all the way to latest?

Reply:
Not at the moment, I am going to attempt this in a separate PR but we anticipate that adopted resource might not work with application autoscaling as is.

Reply:
Oh I did not know you could just disable adoptedResource, that solves one problem for us. We do need to move to the latest version to get namespaced logic in here. 

Will get that second PR out today, I already have the changes.